### DISCUSSION TITLE
Cannot install with Laravel-8

### DISCUSSION THREAD

**Discussion by hanifbd7** [beyondcode/laravel-websockets#894] [General]
> composer require beyondcode/laravel-websockets -W
Using version ^1.12 for beyondcode/laravel-websockets
./composer.json has been updated
Running composer update beyondcode/laravel-websockets --with-all-dependencies
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires laravel/framework ^8.12, found laravel/framework[v8.12.0, ..., 8.x-dev] but it conflicts 
with another require.
  Problem 2
    - livewire/livewire is locked to version v2.5.5 and an update of this package was not requested.
    - livewire/livewire v2.5.5 requires illuminate/validation ^7.0|^8.0 -> found illuminate/validation[v7.0.0, ..., 7.x-dev, v8.0.0, ..., 8.x-dev] but it conflicts with another require.
  Problem 3
    - spatie/laravel-permission is locked to version 4.3.0 and an update of this package was not requested.
    - spatie/laravel-permission 4.3.0 requires illuminate/auth ^6.0|^7.0|^8.0 -> found illuminate/auth[v6.0.0, ..., 6.x-dev, v7.0.0, ..., 7.x-dev, v8.0.0, ..., 8.x-dev] but it conflicts with another require.

**Comment by PaolaRuby** [comment]
**--with-all-dependencies**
Read again

**Reply by hanifbd7** [reply]
Solved by
1.  Adding  "beyondcode/laravel-websockets": "^1.12" in require section of composer. json
2. composer update

**Reply by PaolaRuby** [reply]
It's the same as 
**composer require beyondcode/laravel-websockets --with-all-dependencies**
